src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -1,4 +1,49 @@
-const convertInclude = ({ href, url, type, ...rest }) => {
+import 'isomorphic-fetch';
+import nodeLogger from '#lib/logger.node';
+
+const logger = nodeLogger(__filename);
+
+// this defaults all VJ and IDT2 includes to the specified urls for now
+// should be removed once mozart routes have been created
+const includeUrls = {
+  vj: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/vj.html',
+  idt2: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/idt2.html',
+  idt1: 'https://simorgh-include-test.s3-eu-west-1.amazonaws.com/idt1.html',
+};
+
+export const encodeHTML = str =>
+  String(str)
+    .replace(/&/g, '&amp;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/"/g, '&quot;')
+    .replace(/'/g, '&#39;');
```

Comment:
Can we revisit the need for this conversion and check if there is any alternative?